docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/configure-aws.adoc

```diff
@@ -82,27 +98,24 @@ You can use wildcards to specify a range of tags in both keys and values followi
 "*abcd*"
 ----
 
-.. *Network resources*: Configure custom network resources for agentless scanning. When using custom network resources, Prisma Cloud assumes those resources have a path to communicate outbound data to the Prisma Cloud backend, as explained in the xref:../agentless-scanning.adoc#networking-infrastructure[networking infrastructure section].
+[#aws-agentless-network]
+==== Network Resources
 
-... *Subnet name*: the name tag of the subnet resource in your AWS account. If the subnet allows auto-assignment of public IPs, a public IP will be attached to the scanner instance. Subnet names should be identical and unique across all regions.
+* *Network resources*: Configure custom network resources for agentless scanning. When using custom network resources, Prisma Cloud assumes those resources have a path to communicate outbound data to the Prisma Cloud backend, as explained in the xref:../agentless-scanning.adoc#networking-infrastructure[networking infrastructure section].
 
-... *Security group name*: the name of the security group resource in your AWS account. Security group names should be identical and unique across all regions.
+** *Subnet name*: the name tag of the subnet resource in your AWS account. If the subnet allows auto-assignment of public IPs, a public IP will be attached to the scanner instance. Subnet names should be identical and unique across all regions.
+
+** *Security group name*: the name of the security group resource in your AWS account. Security group names should be identical and unique across all regions.
+
+The following combinations are possible for the network resources.
 
-... The following combinations are possible for the network resources.
-+
 * If you leave both of the following fields blank, Prisma Cloud creates all required network resources and uses a public IP as explained in the xref:../agentless-scanning.adoc#networking-infrastructure[networking infrastructure section].
```

Comment:
`following fields` - doesn't make sense, since the fields appear above this.
Maybe `above fields` ?

Reply:
I think removing the refernce works best. WDYT?